### PR TITLE
Adding functionality to EETest.

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
This PR creates test for E2E testing (re-implementing from old che-functional tests).

### What issues does this PR fix or reference?
https://github.com/redhat-developer/che-functional-tests/issues/292

### How have you tested this PR?
Localy against production and prod-preview.

### CODE REVIEW

**Comment by Katka92** [comment]
@rhopp is that check enough there? Should I do "git log" in terminal and compare last commit id with commit id gotten from ref ( ref.getTarget() ) to be sure that the last commits are the same?

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

File: .ci/cico_rhche_prcheck.sh [hunk 1]
```diff
@@ -84,7 +84,8 @@ fi
 set -x
 
 # Getting core repos ready
-rpm -iUvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+#rpm -iUvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```
Comment (ScrewTSW) [inline .ci/cico_rhche_prcheck.sh]: unnecessary comment.

File: functional-tests/src/test/java/com/redhat/che/functional/tests/EETest.java [hunk 2]
```diff
@@ -14,31 +14,111 @@
...
               + " was successfully injected. ");
+      ide.open(workspace);
+      ide.waitOpenedWorkspaceIsReadyToUse();
     } catch (ExecutionException | InterruptedException e) {
       LOG.error(
           "Could not obtain workspace name and id - worskape was probably not successfully injected.");
+      throw e;
+    } catch (Exception e) {
+      LOG.error("Could not open workspace IDE.");
       e.printStackTrace();
```
Comment (ScrewTSW) [inline functional-tests/src/test/java/com/redhat/che/functional/tests/EETest.java]: exception not handled :D

**Pull request merged**